Dynamic-Programming/tests/Abbreviation.test.js

```diff
@@ -0,0 +1,20 @@
+import { abbreviation } from '../Abbreviation.js'
+
+describe('Abbreviation', () => {
+  test('it can abbreviate string a to b', () => {
```

Comment:
I'd partition the test cases into positives and negatives (two different `test` blocks), then run the individual tests using a helper function `const expectPositive = (word, abbr) => expect(abbreviation(word, abbr)).toBe(true)` to make it somewhat less verbose (matter of taste though).